src/btree/bt_sync.c

```diff
@@ -297,8 +209,133 @@ __sync_delete_obsolete_ref(WT_SESSION_IMPL *session, WT_REF *ref)
       (void *)ref, tag, obsolete ? "" : "not ",
       __wt_time_point_to_string(
         newest_stop_ts, newest_stop_durable_ts, newest_stop_txn, tp_string));
+    return (0);
+}
 
-err:
+/*
+ * __sync_delete_ondisk_obsolete_ref --
+ *     Check whether the on-disk ref is obsolete according to the newest stop time point and handle
+ *     the obsolete page by either remove it.
```

Comment:
either remove it or what?

Reply:
The comment has to end with `remove`, but it was missed during the refactoring. Fixed now.

Reply:
I adjusted the comment, can you please check again and let me know.